tests/TestCase/Http/ResponseTest.php

```diff
@@ -1491,6 +1491,53 @@ public function testWithCookieObject()
         $this->assertSame($cookie, $new->getCookieCollection()->get('yay'));
     }
 
+    public function testWithoutCookieScalar()
+    {
+        $response = new Response();
+        $response = $response->withCookie('testing', 'abc123');
+        $this->assertEquals('abc123', $response->getCookie('testing')['value']);
+
+        $new = $response->withoutCookie('testing');
+
+        $this->assertNull($response->getCookie('testing')['expire']);
+        $this->assertEquals('1', $new->getCookie('testing')['expire']);
+    }
+
+    public function testWithoutCookieOptions()
+    {
+        $options = [
+            'name' => 'testing',
+            'value' => 'abc123',
+            'domain' => 'cakephp.org',
+            'path' => '/custompath/',
+            'secure' => true,
+            'httpOnly' => true,
+            'expire' => (string)strtotime('+14 days'),
+        ];
+
+        $response = new Response();
+        $response = $response->withCookie('testing', $options);
+        $this->assertEquals($options, $response->getCookie('testing'));
+
+        $new = $response->withoutCookie('testing', $options);
```

Comment:
expire as an option to this method should probably trigger an error, as should 'value'.  At the very least those options should be ignored. The test doesn't cover the value being ignored.